README.md

```diff
@@ -97,6 +104,7 @@ cyclictest supports the following environment variables:
 + stress: choice of false/stress-ng
 + rt_priority: which rt priority is used to run the cyclictest; default 1
 + delay: specify how many seconds to delay before test start; default 0
++ TRACE_THRESHOLD: stop the cyclictest when threshold triggered (in usec); no default
```

Comment:
Please add this to the list of options for the standalone cyclictest described later in this file.

Reply:
Added in latest commit